gym/envs/mujoco/half_cheetah.py

```diff
@@ -8,9 +8,9 @@ def __init__(self):
         utils.EzPickle.__init__(self)
 
     def _step(self, action):
-        xposbefore = self.model.data.qpos[0, 0]
+        xposbefore = self.sim.data.qpos[0]
```

Comment:
My understanding is that `qpos` lists joint angles. In `half_cheetah.xml`, it looks like the first joint is "rootx", a slide joint that should give displacement in the x direction. So `qpos[0]` should give the desired value.